nuxeo-distribution/nuxeo-launcher/src/main/java/org/nuxeo/launcher/NuxeoLauncher.java

```diff
@@ -739,6 +739,9 @@ protected void start(boolean logProcessOutput) throws IOException, InterruptedEx
         startCommand.add(getClassPath());
         startCommand.addAll(getNuxeoProperties());
         startCommand.addAll(getServerProperties());
+        if (strict) {
+            startCommand.add("-Dnuxeo.start.strict");
```

Comment:
I would prefer `-Dnuxeo.start.strict=true` and parse it as a boolean, so that it can be disabled by just changing a value instead of removing the option.

Reply:
I didn't think about the property manipulation. Thanks